src/main/java/educationalproject/programmingstuff/controller/Controller.java

```diff
@@ -1,14 +1,57 @@
 package educationalproject.programmingstuff.controller;
 
+import educationalproject.programmingstuff.model.User;
+import educationalproject.programmingstuff.repository.ItemRepository;
+import educationalproject.programmingstuff.repository.OrderRepository;
+import educationalproject.programmingstuff.repository.UserRepository;
+import educationalproject.programmingstuff.service.TestService;
+import educationalproject.programmingstuff.service.UserServiceImpl;
+import lombok.RequiredArgsConstructor;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @RestController
+@RequiredArgsConstructor
 public class Controller {
 
-    @GetMapping("/test")
-    public String test() {
-        return "Don't bother me";
+    private final UserServiceImpl userService;
```

Comment:
Inject only by interface, not by implementation. In Russia we call it "slabaya svyazannost":)

Reply:
Yep, it's "low cohesion" in English